src/core/app.lua

```diff
@@ -20,6 +20,12 @@ local use_restart = false
 
 test_skipped_code = 43
 
+ingress_packet_drops = {
+   threshold = 100000,
+   wait = 20,
+   last_value = 0,
```

Comment:
but, maybe this is for a followup, we can release on monday like this; dunno.  we'll need to do performance testing too, and that will take a bit of time.